tests/test_cmdline.py

```diff
@@ -497,8 +507,19 @@ def test_list_sources_detailed__in_varied_project__lists_all_files(fake_project)
     ]
     major, minor = sys.version_info[:2]
     expect_pyenv_lines = [
-        f"  {tmp_path}/my_venv/lib/python{major}.{minor}/site-packages "
-        + "(as a source of Python packages)",
+        (
+            "  " + str(tmp_path / "my_venv" / "Lib" / "site-packages")
+            if platform.system() == "Windows"
+            else "  "
+            + str(
+                tmp_path
+                / "my_venv"
+                / "lib"
+                / f"python{major}.{minor}"
+                / "site-packages"
+            )
+        )
+        + " (as a source of Python packages)",
```

Comment:
Ok, time to factor our a helper function, I think:

```python
def site_packages(venv_dir: Path) -> Path:
    if sys.platform.startswith("win"):  # Windows
        return venv_dir / "Lib" / "site-packages"
    else:  # Assume POSIX
        major, minor = sys.version_info[:2]
        return venv_dir / f"lib/python{major}.{minor}/site-packages"
```
Put this near the top of the file (or in `tests/utils.py` to share across multiple tests), and then:

```suggestion
        f"  {site_packages(tmp_path / 'my_venv')} (as a source of Python packages)",
```

Reply:
Updated in 127473a053bfcf991c9a92f78a3e440f6d84cd0e